pythonforandroid/distribution.py

```diff
@@ -190,7 +190,18 @@ def get_distributions(cls, ctx, extra_dist_dirs=[]):
                 dist.recipes = dist_info['recipes']
                 if 'archs' in dist_info:
                     dist.archs = dist_info['archs']
-                dist.ndk_api = dist_info['ndk_api']
+                if 'ndk_api' in dist_info:
+                    dist.ndk_api = dist_info['ndk_api']
+                else:
+                    dist.ndk_api = 0
+                    warning(
+                        "Distribution {distname}: (distdir) has been "
+                        "built with an unknown api target, ignoring it, "
+                        "you might want to delete it".format(
+                            distname=dist.name,
+                            distdir=dist.dist_dir
```

Comment:
`distdir` seems unused here

Reply:
I though it would make it easier for user to remove it… but i can remove it if it's not relevant.

Reply:
No sorry what I mean is the format string seems to add a param that's not used, right?

Reply:
Yea kind of looks like `(distdir)` should be `{distdir}`

Reply:
uh, i misread that >_<, thanks for your patience :laughing: